src/main/java/seedu/intrack/model/internship/Internship.java

```diff
@@ -125,18 +123,18 @@ public boolean equals(Object other) {
         Internship otherInternship = (Internship) other;
         return otherInternship.getName().equals(getName())
                 && otherInternship.getPosition().equals(getPosition())
+                // && otherInternship.getStatus().equals(getStatus())
```

Comment:
Would it be better to also check for equality in status?

Reply:
```suggestion
                && otherInternship.getStatus().equals(getStatus())
```